diff -urN pdflib-4.0.0.org/Makefile.in pdflib-4.0.0/Makefile.in
--- pdflib-4.0.0.org/Makefile.in	Thu May 10 21:46:34 2001
+++ pdflib-4.0.0/Makefile.in	Thu May 10 22:30:44 2001
@@ -10,11 +10,11 @@
 srcdir = @srcdir@
 
 # installation directories
-prefix 		= @prefix@
-exec_prefix 	= @exec_prefix@
-libdir 		= @libdir@
-includedir 	= @includedir@
-bindir      	= @bindir@
+prefix 		= $(DESTDIR)@prefix@
+exec_prefix 	= $(DESTDIR)@exec_prefix@
+libdir 		= $(DESTDIR)@libdir@
+includedir 	= $(DESTDIR)@includedir@
+bindir      	= $(DESTDIR)@bindir@
 
 # ------------------------- default target  -------------------------------
 
@@ -111,6 +111,7 @@
 # --------------------------- install ---------------------------------
 
 install: pdflib_install @BINDINGS_INSTALL@
+	$(INSTALL) -d $(bindir)
 	$(INSTALL) pdflib-config $(bindir)
 
 pdflib_install:
diff -urN pdflib-4.0.0.org/bind/java/Makefile.in pdflib-4.0.0/bind/java/Makefile.in
--- pdflib-4.0.0.org/bind/java/Makefile.in	Thu May 10 21:46:34 2001
+++ pdflib-4.0.0/bind/java/Makefile.in	Thu May 10 22:12:13 2001
@@ -15,8 +15,8 @@
 CC		= @CC@
 RM      	= @RM@
 LIBTOOL		= ../../libtool --silent
-prefix 		= @prefix@
-exec_prefix 	= @exec_prefix@
+prefix 		= $(DESTDIR)@prefix@
+exec_prefix 	= $(DESTDIR)@exec_prefix@
 
 PDFLIBCONVENIENT= @PDFLIBCONVENIENT@
 
@@ -28,7 +28,7 @@
 .c.$(OBJ):
 	$(LIBTOOL) --mode=compile $(CC) -c $(CPPFLAGS) $(DEFINES) $<
 
-LANG_LIBDIR 	= @libdir@
+LANG_LIBDIR 	= $(DESTDIR)@libdir@
 
 SWIG		= swig1.2 -I../../pdflib -java -stat
 SWIG_LIB	= libpdf_java.la
@@ -85,6 +85,7 @@
 	    -dlopen $(SWIG_LIB) java quickreference
 
 install: $(SWIG_LIB) pdflib.jar
+	$(INSTALL) -d $(LANG_LIBDIR)
 	@-if test -f .libs/libpdf_java.jnilib; then		\
 	    $(LIBTOOL) --mode=install $(INSTALL_DATA)		\
 		.libs/libpdf_java.jnilib $(LANG_LIBDIR);	\
diff -urN pdflib-4.0.0.org/bind/perl/Makefile.in pdflib-4.0.0/bind/perl/Makefile.in
--- pdflib-4.0.0.org/bind/perl/Makefile.in	Thu May 10 21:46:34 2001
+++ pdflib-4.0.0/bind/perl/Makefile.in	Thu May 10 22:12:13 2001
@@ -26,8 +26,8 @@
 .c.$(OBJ):
 	$(LIBTOOL) --mode=compile $(CC) -c $(CPPFLAGS) $(DEFINES) $<
 
-LANG_LIBDIR 	= @PERLLIBDIR@
-PERLMODDIR	= @PERLMODDIR@
+LANG_LIBDIR 	= $(DESTDIR)@PERLLIBDIR@
+PERLMODDIR	= $(DESTDIR)@PERLMODDIR@
 PERLBIN		= @PERL@ -I. -I.libs
 
 SWIG		= swig1.2 -I../../pdflib -perl5 -shadow -stat
@@ -60,6 +60,7 @@
 	-$(PERLBIN) quickreference.pl
 
 install: $(SWIG_LIB)
+	$(INSTALL) -d $(LANG_LIBDIR)
 	$(LIBTOOL) --mode=install \
 		$(INSTALL_DATA) $(SWIG_LIB) $(LANG_LIBDIR)
 	$(INSTALL_DATA) pdflib_pl.pm $(LANG_LIBDIR)
diff -urN pdflib-4.0.0.org/bind/python/Makefile.in pdflib-4.0.0/bind/python/Makefile.in
--- pdflib-4.0.0.org/bind/python/Makefile.in	Thu May 10 21:46:35 2001
+++ pdflib-4.0.0/bind/python/Makefile.in	Thu May 10 22:12:13 2001
@@ -26,7 +26,7 @@
 .c.$(OBJ):
 	$(LIBTOOL) --mode=compile $(CC) -c $(CPPFLAGS) $(DEFINES) $<
 
-LANG_LIBDIR 	= @PYTHONLIBDIR@
+LANG_LIBDIR 	= $(DESTDIR)@PYTHONLIBDIR@
 
 SWIG		= swig1.2 -I../../pdflib -python -stat
 SWIG_LIB	= pdflib_py.la
@@ -62,6 +62,7 @@
 	-$(LIBTOOL) --mode=execute -dlopen $(SWIG_LIB) python personalize.py
 
 install: $(SWIG_LIB)
+	$(INSTALL) -d $(LANG_LIBDIR)
 	@-if test -f .libs/libpdflib_py.dylib; then		\
 	    $(INSTALL_DATA) .libs/pdflib_py.so $(LANG_LIBDIR);	\
 	else							\
diff -urN pdflib-4.0.0.org/bind/tcl/Makefile.in pdflib-4.0.0/bind/tcl/Makefile.in
--- pdflib-4.0.0.org/bind/tcl/Makefile.in	Thu May 10 21:46:35 2001
+++ pdflib-4.0.0/bind/tcl/Makefile.in	Thu May 10 22:12:13 2001
@@ -26,8 +26,8 @@
 .c.$(OBJ):
 	$(LIBTOOL) --mode=compile $(CC) -c $(CPPFLAGS) $(DEFINES) $<
 
-LANG_LIBDIR 	= @TCLPACKAGEDIR@
-TCLBIN		= @TCLBIN@
+LANG_LIBDIR 	= $(DESTDIR)@TCLPACKAGEDIR@
+TCLBIN		= $(DESTDIR)@TCLBIN@
 
 SWIG		= swig1.2 -I../../pdflib -tcl -stat
 SWIG_LIB	= pdflib_tcl.la
diff -urN pdflib-4.0.0.org/pdflib/Makefile.in pdflib-4.0.0/pdflib/Makefile.in
--- pdflib-4.0.0.org/pdflib/Makefile.in	Thu May 10 21:46:34 2001
+++ pdflib-4.0.0/pdflib/Makefile.in	Thu May 10 22:12:13 2001
@@ -16,11 +16,11 @@
 INTERNALLIBS	= @INTERNALLIBS@
 
 # installation directories
-prefix 		= @prefix@
-exec_prefix 	= @exec_prefix@
-libdir 		= @libdir@
-includedir 	= @includedir@
-bindir      	= @bindir@
+prefix 		= $(DESTDIR)@prefix@
+exec_prefix 	= $(DESTDIR)@exec_prefix@
+libdir 		= $(DESTDIR)@libdir@
+includedir 	= $(DESTDIR)@includedir@
+bindir      	= $(DESTDIR)@bindir@
 
 TIFFLIBINC	= @TIFFLIBINC@
 ZLIBINC		= @ZLIBINC@
