diff -Nur pdflib-3.02.orig/bind/java/Makefile.in pdflib-3.02/bind/java/Makefile.in
--- pdflib-3.02.orig/bind/java/Makefile.in	Tue Aug 22 03:02:00 2000
+++ pdflib-3.02/bind/java/Makefile.in	Sat Sep 16 06:16:13 2000
@@ -16,8 +16,8 @@
 CC		= @CC@
 RM      	= @RM@
 LIBTOOL		= ../../libtool
-prefix 		= @prefix@
-exec_prefix 	= @exec_prefix@
+prefix 		= $(DESTDIR)@prefix@
+exec_prefix 	= $(DESTDIR)@exec_prefix@
 
 PDFLIBLINK	= @PDFLIBLINK@
 
@@ -29,7 +29,7 @@
 .c.$(OBJ):
 	@$(LIBTOOL) --mode=compile $(CC) -c $(CPPFLAGS) $(DEFINES) $<
 
-LANG_LIBDIR 	= @libdir@
+LANG_LIBDIR 	= $(DESTDIR)@libdir@
 
 SWIG		= swig1.2 -I../../pdflib -java -stat
 SWIG_LIB	= libpdf_java.la
@@ -81,6 +81,7 @@
 	    -dlopen $(SWIG_LIB) java pdfclock
 
 install: $(SWIG_LIB) pdflib.jar
+	install -d $(LANG_LIBDIR)
 	$(LIBTOOL) --mode=install $(INSTALL_DATA) $(SWIG_LIB) $(LANG_LIBDIR)
 	@echo Copy pdflib.jar and javadoc to an appropriate directory
 	@echo and include the pdflib.jar file name in your CLASSPATH.
diff -Nur pdflib-3.02.orig/bind/perl/Makefile.in pdflib-3.02/bind/perl/Makefile.in
--- pdflib-3.02.orig/bind/perl/Makefile.in	Tue Aug 22 03:02:00 2000
+++ pdflib-3.02/bind/perl/Makefile.in	Sat Sep 16 06:16:38 2000
@@ -27,8 +27,8 @@
 .c.$(OBJ):
 	@$(LIBTOOL) --mode=compile $(CC) -c $(CPPFLAGS) $(DEFINES) $<
 
-LANG_LIBDIR 	= @PERLLIBDIR@
-PERLMODDIR	= @PERLMODDIR@
+LANG_LIBDIR 	= $(DESTDIR)@PERLLIBDIR@
+PERLMODDIR	= $(DESTDIR)@PERLMODDIR@
 PERLBIN		= @PERL@
 
 SWIG		= swig1.2 -I../../pdflib -perl5 -shadow -stat
@@ -63,6 +63,7 @@
 	-$(LIBTOOL) --mode=execute -dlopen $(SWIG_LIB) $(PERLBIN) pdfclock.pl
 
 install: $(SWIG_LIB)
+	install -d $(LANG_LIBDIR)
 	$(LIBTOOL) --mode=install \
 		$(INSTALL_DATA) $(SWIG_LIB) $(LANG_LIBDIR)
 	$(INSTALL_DATA) pdflib_pl.pm $(LANG_LIBDIR)
diff -Nur pdflib-3.02.orig/bind/python/Makefile.in pdflib-3.02/bind/python/Makefile.in
--- pdflib-3.02.orig/bind/python/Makefile.in	Tue Aug 22 03:02:00 2000
+++ pdflib-3.02/bind/python/Makefile.in	Sat Sep 16 06:17:05 2000
@@ -19,8 +19,8 @@
 
 PDFLIBLINK	= @PDFLIBLINK@
 
-prefix 		= @prefix@
-exec_prefix 	= @exec_prefix@
+prefix 		= $(DESTDIR)@prefix@
+exec_prefix 	= $(DESTDIR)@exec_prefix@
 
 .SUFFIXES: .c .$(OBJ) .lo
 
@@ -30,7 +30,7 @@
 .c.$(OBJ):
 	@$(LIBTOOL) --mode=compile $(CC) -c $(CPPFLAGS) $(DEFINES) $<
 
-LANG_LIBDIR 	= @PYTHONLIBDIR@
+LANG_LIBDIR 	= $(DESTDIR)@PYTHONLIBDIR@
 
 SWIG		= swig1.2 -I../../pdflib -python -stat
 SWIG_LIB	= pdflib_py.la
@@ -65,6 +65,7 @@
 	-$(LIBTOOL) --mode=execute -dlopen $(SWIG_LIB) python pdfclock.py
 
 install: $(SWIG_LIB)
+	install -d $(LANG_LIBDIR)
 	$(LIBTOOL) --mode=install \
 		$(INSTALL_DATA) $(SWIG_LIB) $(LANG_LIBDIR)
 
diff -Nur pdflib-3.02.orig/bind/tcl/Makefile.in pdflib-3.02/bind/tcl/Makefile.in
--- pdflib-3.02.orig/bind/tcl/Makefile.in	Tue Aug 22 03:02:00 2000
+++ pdflib-3.02/bind/tcl/Makefile.in	Sat Sep 16 06:15:49 2000
@@ -27,8 +27,8 @@
 .c.$(OBJ):
 	@$(LIBTOOL) --mode=compile $(CC) -c $(CPPFLAGS) $(DEFINES) $<
 
-LANG_LIBDIR 	= @TCLPACKAGEDIR@
-TCLBIN		= @TCLBIN@
+LANG_LIBDIR 	= $(DESTDIR)@TCLPACKAGEDIR@
+TCLBIN		= $(DESTDIR)@TCLBIN@
 
 SWIG		= swig1.2 -I../../pdflib -tcl -stat
 SWIG_LIB	= pdflib_tcl.la
diff -Nur pdflib-3.02.orig/pdflib/Makefile.in pdflib-3.02/pdflib/Makefile.in
--- pdflib-3.02.orig/pdflib/Makefile.in	Tue Aug 22 03:02:00 2000
+++ pdflib-3.02/pdflib/Makefile.in	Sat Sep 16 06:15:49 2000
@@ -17,12 +17,12 @@
 LIBTOOL		= ../libtool
 
 # installation directories
-prefix = @prefix@
-exec_prefix = @exec_prefix@
+prefix = $(DESTDIR)@prefix@
+exec_prefix = $(DESTDIR)@exec_prefix@
 
 # directories for the final library and include files for PDFlib users
-libdir = @libdir@
-includedir = @includedir@
+libdir = $(DESTDIR)@libdir@
+includedir = $(DESTDIR)@includedir@
 
 TIFFLIBINC	= @TIFFLIBINC@
 ZLIBINC		= @ZLIBINC@
